tests/foreman/ui/test_syncplan.py

```diff
@@ -118,6 +122,13 @@ def test_positive_end_to_end(session):
         session.syncplan.update(plan_name, {'details.description': new_description})
         syncplan_values = session.syncplan.read(plan_name)
         assert syncplan_values['details']['description'] == new_description
+        # Create and add two products to sync plan
+        for _ in range(0, 2):
```

Comment:
FYI, you can skip starting position when it's 0:
```suggestion
        for _ in range(2):
```

Reply:
yeah, OK